### ISSUE TITLE
Can't receive blur on ConsoleWindowClass window titlebars on Windows 11 22621

### ISSUE THREAD

**Issue by UnclWish** [MicaForEveryone/MicaForEveryone#196]
How can I achieve blur effect for window titlebar on ConsoleWindowClass windows on Windows 11 22621?
I saw screenshots from peoples how can make it with MFE, but for me it doesnn't work. No blur.
I use this config:

Global {
  TitleBarColor             = System  # available modes: Default, System, Light, Dark (case sensitive)
  BackdropPreference        = Mica    # available modes: Default, None, Mica, Acrylic, Tabbed
  CornerPreference          = Default # available modes: Default, Square, Rounded, RoundedSmall
  ExtendFrameIntoClientArea = False   # enable to apply backdrop on background of apps (not recommended)
  EnableBlurBehind          = False   # make window translucent (not recommended)
}

# Don't touch explorer and mspaint
Process: "explorer" {
  TitleBarColor = Default
  BackdropPreference = Mica
  CornerPreference = Default
  ExtendFrameIntoClientArea = False
  EnableBlurBehind = False
}
Process: "mspaint" {
  TitleBarColor = Default
  BackdropPreference = Mica
  CornerPreference = Default
  ExtendFrameIntoClientArea = False
  EnableBlurBehind = False
}

# Exclude OpenConsole due to #120.
# TODO: Improve the window exclusion check within the MFE source code.
Process: "OpenConsole" {
  TitleBarColor = Default
  BackdropPreference = Mica
  CornerPreference = RoundedSmall
  ExtendFrameIntoClientArea = True
  EnableBlurBehind = False
}

# Apply Mica background to Console Host window
Class: "ConsoleWindowClass" {
  TitleBarColor = Default
  BackdropPreference = Mica
  CornerPreference = RoundedSmall
  ExtendFrameIntoClientArea = False
  EnableBlurBehind = False
}

**Comment by xmine64** [comment]
Enable extend frame into client area and blur behind.

**Comment by MuscularPuky** [comment]
yes, because of HDR. i have that too. as i know that's windows problem so wont be able to fix it–so sad

ConsoleWindowClass doesnt have titlebar. if you make blurred only titlebar, you shuold specify the process, not class.

note: it doesnt apply to elevated one. running mfe as administrator will solve this but still doesnt to "system". maybe needs psexec?

**Comment by Battler624** [comment]
is there any development on this?
